### PR TITLE
Unable to request tokens

### LINKED ISSUE #4979: Unable to request tokens
### Expected behavior

- Fetch network/app supported tokens for request tokens

### Actual behavior

image

### Steps to reproduce

Request tokens from an un initialised account

### Which version(s) does this affect? (Environment, OS, etc...)
3.0

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #4979, #4948

### How was it solved?

:bug: Fix request token

### How was it tested?

- Unit tests
- Manually

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -5,6 +5,8 @@
 .gitattributes
```
Comment (oskarleonard) [inline .prettierignore:5]: Not related to this line/file:
Changing back to Lisk application gives error

image

File: src/modules/token/fungible/hooks/queries/useNetworkSupportedTokens.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    config: { params: { chainID: application?.chainID } },
+    client: client.current,
+  });
+  const isLoading = tokensSupported.isLoading || appsMetaTokens.isLoading;
+  const isFetched = tokensSupported.isFetched && appsMetaTokens.isFetched;
+  const isError = tokensSupported.isError || appsMetaTokens.isError;
+
+  return useMemo(() => {
+    let tokens = [];
+    console.log('.>>>>', isSupportAllTokens);
```
Comment (oskarleonard) [inline src/modules/token/fungible/hooks/queries/useNetworkSupportedTokens.js]: remove?

File: .prettierignore (hunk 1 above)
Comment (ManuGowda) [reply .prettierignore:5]: This is a valid error, in the past we were showing empty state when we had network error, now it should there is a error.

Comment (oskarleonard) [reply .prettierignore:5]: But first time i open the modal it works. But maybe this is related to the network being flunky?

Comment (ManuGowda) [reply .prettierignore:5]: yea.. if the network is flaky it will show the error, and if it fetches the metadata then it shouldn't show error.

**Review by oskarleonard — DISMISSED** [review]

**Review by oskarleonard — APPROVED** [review]

**Review by eniolam1000752 — APPROVED** [review]

**Pull request merged**